scripts/g_r.py

```diff
@@ -0,0 +1,60 @@
+import datetime
+import pandas
+from collections import Counter
+from sqlalchemy import create_engine
+engine = create_engine('mysql://root@localhost:3306/pantherbot_test', echo=False)
+
+def g_r(response, args):
```

Comment:
Code should be self documenting, you could add a comment, but its better to have your function describe what it does in a succinct way